main/index.js

```diff
@@ -38,7 +42,8 @@ const { setup: setupDialogs } = require('./dialog')
 const inTest = (process.env.NODE_ENV === 'test')
 const isDev = !app.isPackaged && !inTest
 
-console.log('Filecoin Station build version:', BUILD_VERSION)
+log.info('Filecoin Station build version:', BUILD_VERSION, isDev ? '[DEV]' : '', inTest ? '[TEST]' : '')
+log.info('Running on %s %s version %s', os.platform(), os.arch(), os.release())
```

Comment:
I am not sure which APIs are better to use. Do you have any opinion on this?

- `os.arch()` vs `os.machine()`
- `os.platform()` vs `os.type()`

The former APIs describe what is the target for which Node.js was built. The latter APIs make sys calls to determine the value.

Example output:

```
> console.log({arch: os.arch(), machine: os.machine(), platform: os.platform(), type: os.type()})
{ arch: 'arm64', machine: 'arm64', platform: 'darwin', type: 'Darwin' }
```

Node.js API docs:

https://nodejs.org/dist/latest-v18.x/docs/api/os.html#osarch
> Returns the operating system CPU architecture for which the Node.js binary was compiled. Possible values are 'arm', 'arm64', 'ia32', 'mips', 'mipsel', 'ppc', 'ppc64', 's390', 's390x', and 'x64'.

https://nodejs.org/dist/latest-v18.x/docs/api/os.html#osmachine
> Returns the machine type as a string, such as arm, aarch64, mips, mips64, ppc64, ppc64le, s390, s390x, i386, i686, x86_64.

https://nodejs.org/dist/latest-v18.x/docs/api/os.html#osplatform
> Returns a string identifying the operating system platform for which the Node.js binary was compiled. The value is set at compile time. Possible values are 'aix', 'darwin', 'freebsd','linux', 'openbsd', 'sunos', and 'win32'.

https://nodejs.org/dist/latest-v18.x/docs/api/os.html#ostype
> Returns the operating system name as returned by [uname(3)](https://linux.die.net/man/3/uname). For example, it returns 'Linux' on Linux, 'Darwin' on macOS, and 'Windows_NT' on Windows.

Reply:
Let's print both. The build target can be different from the computer architecture, e.g. when running Darwin x86_64 binaries on arm64 via Rosetta.

Reply:
Unfortunately, `os.machine()` has been added in Node.js v18, which is not available in Electron yet. Hopefully, we will get this in Feb 2023 - see https://github.com/electron/electron/pull/36356 and https://www.electronjs.org/docs/latest/tutorial/electron-timelines

<img width="724" alt="Screenshot 2022-12-14 at 15 03 27" src="https://user-images.githubusercontent.com/1140553/207615782-779f2176-fa97-41ac-a431-a851a1ac588d.png">